CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Change Log
+
+
+## v0.0.2
```

Comment:
See demos-europe/demosplan-ui#8, this is technically incorrect at this stage but not a blocker.

Reply:
I would like to lern how to maintain a good changelog. i hope we can get better here soon